File: app/src/main/java/ys/moire/infra/prefs/Prefs.java
package ys.moire.infra.prefs;

/**
 * Prefs.
 */

public interface Prefs {

    void put(final String key, final String value);

    void put(final String key, final int value);

    String get(final String key, final String defValue);

    int get(final String key, final int defValue);
}

File: app/src/main/java/ys/moire/infra/prefs/Prefs.kt
package ys.moire.infra.prefs

/**
 * Prefs.
 */

interface Prefs {

    fun put(key: String, value: String)

    fun put(key: String, value: Int)

    operator fun get(key: String, defValue: String): String?

    operator fun get(key: String, defValue: Int): Int
}
